### PR TITLE
Feature/user field configuration

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/FORMS-671

* Adds OS2Forms Organisation OpenID Connect module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ drush pm:enable os2forms_organisation
 
 Edit settings on `/admin/os2forms_organisation/settings`.
 
+**NOTE** This module creates and uses a user field called `field_organisation_user_id`.
+It is up to the using party to ensure it is filled out with the SF1500
+organisation  `bruger` ID. If it is not, nothing is displayed.
```
Comment (rimi-itk) [inline README.md]: Trim space after “organisation”. Your text editor should have a “Fill paragraph” command (like https://www.emacswiki.org/emacs/FillParagraph, say) that can help you clean this up.

“bruger” should probably be quoted with regular quotes, i.e. “bruger”, and not quoted as inline code.

Can we provide a link to more details on “the SF1500 organisation  `bruger` ID”?

File: modules/os2forms_organisation_openid_connect/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# OS2Forms Organisation OpenID Connect
+
+Module for OS2Forms Organisation OpenID Connect.
```
Comment (rimi-itk) [inline modules/os2forms_organisation_openid_connect/README.md]: This is basically just repeating the heading. Tell a bit about what this module actually does.

File: modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+name: OS2Forms Organisation OpenID Connect
+type: module
+description: Module for OS2Forms Organisation OpenID Connect.
+package: OS2Forms
+core_version_requirement: ^9
```
Comment (rimi-itk) [inline modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml:5]: This module should probably depend on the OS2Forms Organisation module.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jekuaitk) [reply README.md]: Updated.

File: modules/os2forms_organisation_openid_connect/README.md (hunk 2 above)
Comment (jekuaitk) [reply modules/os2forms_organisation_openid_connect/README.md]: Updated.

File: modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml (hunk 3 above)
Comment (jekuaitk) [reply modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml:5]: Seems reasonable. Added dependency.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**